frontend/src/components/Sidebar/SidebarStyles.js

```diff
@@ -1,12 +1,37 @@
 import styled from 'styled-components';
-import { Navbar } from 'react-bootstrap';
+import { Navbar, Nav } from 'react-bootstrap';
+
 
 export const SidebarEdit = styled(Navbar)`
     width: var(--sidebar-width);
     height: 100vh;
-    background-color: purple; // TODO: use figma color, this is for testing only
+    background-color: var(--dark-blue); // TODO: use figma color, this is for testing only
```

Comment:
Można już ten komentarz wyrzucić
```suggestion
    background-color: var(--dark-blue);
```